src/pages/_app.tsx

```diff
@@ -5,21 +5,20 @@
 import {useEffect} from 'react';
 import {AppProps} from 'next/app';
 import {useRouter} from 'next/router';
-import {ga} from '../utils/analytics';
 
 import '@docsearch/css';
 import '../styles/algolia.css';
 import '../styles/index.css';
 import '../styles/sandpack.css';
 
 if (typeof window !== 'undefined') {
-  if (process.env.NODE_ENV === 'production') {
```

Comment:
Why was this check removed? Does this mean that PR preview deploys and locally hosted environments will now send GA4 events?

Reply:
This check was for the two deprecated ga calls that were also removed so it's no longer needed